spec/rspec/core/example_spec.rb

```diff
@@ -58,6 +58,21 @@ def metadata_hash(*args)
     end
   end
 
+  it "captures example timing even for exceptions unhandled by RSpec" do
+    begin
+      avoided = RSpec::Support::AllExceptionsExceptOnesWeMustNotRescue::AVOID_RESCUING
+      unhandled = Class.new(StandardError)
+      avoided << unhandled
```

Comment:
Yep, it is and was super weird.  I still don't understand it.  1.8.7 has weird bugs that make things hard to explain.